barretenberg/sol/src/honk/Transcript.sol

```diff
@@ -208,4 +213,137 @@ library TranscriptLib {
         zeromorphX = FrLib.fromBytes32(keccak256(abi.encodePacked(buf)));
         zeromorphZ = FrLib.fromBytes32(keccak256(abi.encodePacked(zeromorphX)));
     }
+
+    // TODO: mod q proof points
+    // TODO: Preprocess all of the memory locations
+    // TODO: Adjust proof point serde away from poseidon forced field elements
+    // TODO: move this back to probably each instance to avoid dynamic init of arrays in the Transcript Lib
+    function loadProof(bytes calldata proof) external view returns (Honk.Proof memory) {
```

Comment:
Moved loadProof in Transcript to make my life a bit easier when I have to change the proof content for Gemini and test. The only drawback is a dynamically sized array (but a tiny one). In the copied contract in honk_contract.hpp the loadProof function stayed unchanged